<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Pavan" ignored="0" name="Default test(failed)(failed)(failed)(failed)" tests="2" failures="2" timestamp="2022-04-12T11:08:39 IST" time="38.403" errors="0">
  <testcase name="loginTest" time="5.976" classname="com.shop.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at com.shop.testcases.LoginPageTest.loginTest(LoginPageTest.java:44)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="6.969" classname="com.shop.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at com.shop.testcases.LoginPageTest.loginTest(LoginPageTest.java:44)
at java.util.ArrayList.forEach(ArrayList.java:1249)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test(failed)(failed)(failed)(failed) -->
